tests/io/test_cameras.py

```diff
@@ -590,6 +606,20 @@ def test_instance_group(
             name="test_instance_group",
             name_registry={},
         )
+    instance_by_camera = {
+        cam: instance_group.get_instance(cam) for cam in instance_group.cameras
+    }
+    instance_group_from_dict = InstanceGroup.from_instance_by_camcorder_dict(
+        instance_by_camcorder=instance_by_camera,
+        name="test_instance_group",
+        name_registry={},
+        score=0.5,
+    )
+    assert instance_group_from_dict.score == 0.5
+    # The score of instances will NOT be updated on initialization.
+    for instance in instance_group_from_dict.instances:
+        if isinstance(instance, PredictedInstance):
+            assert instance.score != instance_group_from_dict.score
 
     # Test `__repr__`
     print(instance_group)
```

Comment:
did you mean to leave this?

Reply:
Yes, the `__repr__` method is used when calling `print`